weather_json/extract_code.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <fstream>
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+std::string removeQuotes(const std::string& str) {
+    std::string result = str;
+    result.erase(std::remove(result.begin(), result.end(), '\"'), result.end());
+    return result;
+}
+
+void extractCodeIcon(const json& j, std::ofstream& outputFile) {
+    for (const auto& item : j) {
```

Comment:
I forgot to notify it to you... 🙇‍♀️
This file was used to extract necessary informations (weather codes used by this API) from json file to define weather types. 
I just kept it here because I thought it'll be useful for JSON file parsing in our futur projects (with an external library) 😺 So, we should not push to school's git repo.